src/tests/test_merged_routes.py

```diff
@@ -12,41 +12,22 @@
 
 @freeze_time("2022-01-01")
 def test_read_one_gsp(db_session, api_client):
```

Comment:
hello @peterdudfield, here's my second question about the tests for the gsp forecast route. `test_read_one_gsp` here also throws an error, and I'm not sure of the purpose of the test. And my code is wrong with `_ = ForecastValue(**response.json())`.  

Thanks in advance for any guidance.